Oshanimi/SigninToScoring.feature

```diff
@@ -0,0 +1,140 @@
+Feature: 1.1 Sign in with user's Google Account
+
+Background: 
+	Given user is on sign in 'https://qa.ddso-spot.quantori.com/'
+	And user clicks on the 'signin page' button
+		
+Scenario Outline: 1.1_01 Sign in with invalid password
+	Given 'Choose an option' dialog box is displayed
+	And user clicks on 'Use another account' button
+	When user fills the account 'email text field with value 'beatricek00@gmail.com'
+	And user clicks the 'Next' button
+	And user fills an incorrect password text field with value 'incorrect.password'
+	And user clicks the 'Next' button
+	Then a text 'Can't login! Wrong email or password.' 
+
+Scenario Outline: 1.1_02 Sign in with no registered google account
+	Given 'Choose an option' dialog box is displayed
+	And user clicks on 'Use another account'button
+	When user fills the account 'email or phone' text field with value 'incorrect@yahoo.com'
+	And user clicks the 'Next' button
+	Then the 'email textfield'is marked red. 
+	And a text 'Couldn't find your Google account' 
+
+Scenario Outline: 1.1_03 Sign in with empty 'email' text field
+	Given 'Choose an option' dialog box is displayed
+	And user clicks on 'Use another account' button
+	When user leaves the account 'email or phone' textbox empty
+	And user clicks the 'Next' button
+	Then the 'email textfield'is marked red. 
+	And 'Enter email or phone number' appears in the validation errors region
+
+Scenario Outline: 1.1_04 Sign in with empty 'password' text field
+	Given 'Choose an option' dialog box is displayed
+	And user clicks on 'Use another account' button
+	When user fills the account 'email or phone' text field with a valid value 'beatricek006@gmail.com'
+	But user leaves the password text field empty
+	And user clicks the 'Next' button
+	Then the 'password' text field is marked red. 
+	And 'Enter a password' appears in the validation errors region
+
+Scenario: 1.1_05 Sign in with invalid phone number
+	Given 'Choose an option' dialog box is displayed
+	And user clicks on 'Use another account' button
+	When user fills the account 'email or phone' text field 
+	And user clicks the 'Next' button
+	Then the 'email textfield'is marked red. 
+	And 'Enter email or phone number' appears in the validation errors region
+		
+		Examples: 
+			| Invalid phone number format | type                             |
+			| +79836120001                | Not registered on google account |
+			| +798361200                  | Incomplete phone number          |
+
+	
+Scenario Outline: 1.1_06 Sign in with valid email account 
+	Given 'Choose an option' dialog box is displayed
+	And user clicks on 'Use another account' button
+	When user fills the account email textbox with value 'beatricek006@gmail.com' 
```

Comment:
@shepelevaolga I would really appreciate her opinion here as well as to what is expected. I just think since it is exploratory and the scenarios has been stated.

Reply:
p.s. this quaestion I have  in relation to all scenarios where account credentials are involved

Reply:
@AlexandraPopovaz  Okay. Thanks for the clarification